understanding/21/target-size-enhanced.html

```diff
@@ -7,6 +7,17 @@
 	</head>
 	<body>
 		<h1>Understanding SC 2.5.5 Target Size (Enhanced)</h1>
+
+	<section id="brief">
+		<h2>In brief</h2>
+		<dl>
+			<dt>Objective</dt><dd>Controls can be operated more easily, especially on touch screens</dd>
+			<dt>Author task</dt><dd>Make custom targets at least 44 by 44 pixels big</dd>
```

Comment:
big/in size

Reply:
Yeah, I ventured "big" just to see if I could somehow convey the idea of it being 'bigger', but I agree it read awkwardly If "big" isn't working, I think the cleaner thing is just end after "pixels"
> Make custom targets at least 44 by 44 pixels